client/src/sass/include/_variables.scss

```diff
@@ -93,6 +97,9 @@ $variables: (
 
   --inputBackgroundColor: var(--inputBackgroundColor),
   --inputPlaceholderColor: var(--inputPlaceholderColor),
+  --inputBorderColor: var(--inputBorderColor),
```

Comment:
We have plenty of other components which this added variable won't impact. I suggest you directly use the variable instead of the css custom variable, and rather make that change in a separate PR (though this really isn't a priority… :))

Reply:
Ok ! I put the sass variable in the component instead and removed added css custom / sass variables in `_mixins.scss` and `_variables.scss`